typedb/connection/cluster/server_client.py

```diff
@@ -66,9 +66,13 @@ def close(self) -> None:
 
 
 class _CredentialAuth(grpc.AuthMetadataPlugin):
-    def __init__(self, username, password):
-        self._username = username
-        self._password = password
+    def __init__(self, credential, token):
```

Comment:
I know this is gRPC's fault and not ours, but how about we diverge from our established principle of following gRPC's naming in this case? The fact that the parameter named `credential` is in fact a `Callable` is... very off-putting.
```suggestion
    def __init__(self, credential_fn, token_fn):
```

Reply:
(Also, if you happen to know the correct type hints, that would be a bonus, but it's no big deal if you don't!)